tests/data/data_generator.py

```diff
@@ -8,13 +8,18 @@
 """
 
 import os
-import platform
-from worldengine.plates import _plates_simulation
+from worldengine.plates import world_gen, _plates_simulation
 
 
 def main(tests_data_dir):
-    w = _plates_simulation("Foo", 300, 200, 279)
-    w.to_pickle_file("%s/py%s_plates_279.world" % (tests_data_dir, platform.python_version_tuple()[0]))
+    w = world_gen("seed_28070", 300, 200, 28070)
+    w.protobuf_to_file("%s/seed_28070.world" % tests_data_dir)
+
+    # w = world_gen("seed_48956", 450, 300, 48956)
+    # w.protobuf_to_file("%s/seed_48956.world" % tests_data_dir)
+
+    # w = world_gen("biome_test", 128, 128, 6908)
+    # w.protobuf_to_file("%s/biome_test.world" % tests_data_dir)
```

Comment:
I started adding that code so `data_generator.py` would be able to generate all data at once and I wouldn't have to use an external script. Then I starting removing references to all but one world and I wasn't sure if these specific worlds would maybe be needed again in case the switch to a single world was a bad idea. If everything else is fine, I guess I should remove the commented code.